### DISCUSSION TITLE
Message error:  Property error does not exisit

### DISCUSSION THREAD

**Discussion by gittrolo** [quasarframework/quasar#9114] [General - Components / Directives / etc]
Why does the following error appear?
![image](https://user-images.githubusercontent.com/52664193/116684398-36492180-a9b1-11eb-8c02-67b92998ee41.png)

The code runs smoothly

**Comment by ontwikkelfabriek** [comment]
See this thread on the Vetur github:
https://github.com/vuejs/vetur/issues/1414

and a 'solution' if you are using Typescipt:
https://github.com/vuejs/vetur/issues/1754
